brands/office/insurance.json

```
@@ -725,5 +725,15 @@
       "name:fa": "بیمه ایران",
       "office": "insurance"
     }
+  },
+   "office/insurance|ENSA": {
+    "countryCodes": ["ao"],
+    "tags": {
+      "brand": "ENSA – Seguros de Angola S.A.",
+      "brand:wikidata": "Q5323726",
+      "brand:wikipedia": "en:ENSA_–_Seguros_de_Angola",
```

Need to switch underscores for spaces in the `brand:wikipedia` tag..
```suggestion
      "brand:wikipedia": "en:ENSA – Seguros de Angola",
```